src/main/scala/fi/oph/koski/suoritusjako/aktiivisetjapaattyneetopinnot/AktiivisetJaPaattyneetOpinnotService.scala

```diff
@@ -8,7 +8,7 @@ import fi.oph.koski.log._
 import fi.oph.koski.schema
 import fi.oph.koski.suoritusjako.common.{OpiskeluoikeusFacade, RawOppija}
 
-class AktiivisetJaPaattyneetOpinnotService(application: KoskiApplication) extends GlobalExecutionContext with Logging {
+class AktiivisetJaPäättyneetOpinnotService(application: KoskiApplication) extends GlobalExecutionContext with Logging {
```

Comment:
Mitä ajattelet, voisiko myös tiedostojen nimet muuttaa vastaamaan luokan nimeä eli lisätä skandit niihinkin? Näköjään Koskessa on ainakin aiemmin käytetty skandeja myös tiedostojen nimissä.

Reply:
On käytetty paikoin, mutta ei pitäisi käyttää. Ne on todella ärsyttäviä varsinkin komentorivi-git:in kanssa.